src/main/java/Duke/Command.java

```diff
@@ -0,0 +1,142 @@
+            break;
+        case (DELETE_COMMAND):
+            deleteTask(taskListInput, uiInput, queryTaskNumberText);
+            break;
+        default:
+            insertNewTask(taskListInput, uiInput, tokenizedInput);
+            break;
+        }
+    }
+    private void insertNewTask(TaskList listInput, Ui uiInput, String[] tokenizedInput) throws
+            DukeException{
+        Task newTask = null;
+        switch (tokenizedInput[0]) {
+        case (TODO_COMMAND):
+            newTask = new ToDo(taskDescriptionRemarksFieldsInput[0], taskDescriptionRemarksFieldsInput[1]);
+            break;
+        case (DEADLINE_COMMAND):
+            newTask = new Deadline(taskDescriptionRemarksFieldsInput[0], taskDescriptionRemarksFieldsInput[1]);
+            break;
+        case (EVENT_COMMAND):
+            newTask = new Event(taskDescriptionRemarksFieldsInput[0], taskDescriptionRemarksFieldsInput[1]);
+            break;
+        default:
+            uiInput.displayErrorMessage(INVALID_COMMAND_MESSAGE);
+            //break;
+        }
+
+        listInput.addTask(newTask);
+        String taskAddedMessage = messageContainer.getTaskAddedMessage(newTask, listInput);
+        System.out.println(taskAddedMessage);
+    }
+    public void printTaskList(TaskList listInput, Ui uiInput) {
+        String taskListPrintOutput = new String();
+
+        //if list empty, inform user and await next command
+        if (listInput.getTaskCount() == 0) {
+            uiInput.displayErrorMessage(EMPTY_LIST_ERROR);
+            return;
+        }
+        //if list non-empty, print out all existing tasks
+        for (int i = 0; i < listInput.getTaskCount(); i++) {
+            taskListPrintOutput += "\t" + Integer.toString(i + 1) + "."
+                    + listInput.getTaskList().get(i).toString() +"\n";
+        }
+        System.out.println(taskListPrintOutput);
+        return;
+    }
+
+    public void taskIsDone(TaskList listInput, Ui uiInput, String taskNumberInput) throws DukeException {
+        int queryNumber;
+        try {
+            queryNumber = Integer.parseInt(taskNumberInput);
+        } catch (NumberFormatException e) {
+            //throw error if taskNumber is a string eg. "remove foo" OR whitespaces only
+            throw new DukeException(INVALID_TASK_NUMBER);
+        }
+        boolean isOutOfRange = queryNumber < 1 || queryNumber > listInput.getTaskCount();
+
+        //handle case where user inputs non-existing task number to mark as done
+        if (isOutOfRange){
+            uiInput.displayErrorMessage(TASK_NOT_FOUND);
+            return;
+        }
+        //handle case where user tries to mark as done an already completed task
+       /* boolean isTaskAlreadyDone = listInput.getTaskDoneStatus(queryNumber-1);
+
+        if (isTaskAlreadyDone) {
+            uiInput.displayErrorMessage(TASK_ALREADY_COMPLETED_ERROR_MESSAGE);
+            return;
+        }*/
```

Comment:
Please try to remove commented out code portions. Please check for the rest of the classes to remove all other commented out code portions.